### PR TITLE
Updating testing guide with few additions

### PR DESCRIPTION
**Pull request by agullon**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -132,7 +133,7 @@ Let’s take a look at how to run the chaos scenarios on your OpenShift clusters
 - Zone Outages ([Documentation](https://github.com/cloud-bulldozer/kraken-hub/blob/main/docs/zone-outages.md))
   - Creates outage of availability zone(s) in a targeted region in the public cloud where the OpenShift cluster is running by tweaking the network acl of the zone to simulate the failure and that in turn will stop both ingress and egress traffic from all the nodes in a particular zone for the specified duration and reverts it back to the previous state
     - Helps understand the impact on both Kubernetes/OpenShift control plane as well as applications, services running on the worker nodes in that zone.
-    - Currently only set up for AWS cloud platform
+    - Currently only set up for AWS cloud platform: 1 VPC and multiples subnets within the VPC can be specify
```
Comment (chaitanyaenr) [inline docs/index.md:136]: s/specify/specified

File: docs/index.md [hunk 2]
```diff
@@ -162,8 +163,8 @@ Let’s take a look at how to run the chaos scenarios on your OpenShift clusters
   - Delete namespaces for the specified duration
     - Helps understand the  impact on other components and test/improve recovery time of the components in the targeted namespace
 
-- Persistent volume fill ([Documentation]())
-  - Fills up the persistent volumes used by the pod for the specified duration
+- Persistent volume fill ([Documentation](https://github.com/cloud-bulldozer/kraken-hub/blob/main/docs/pvc-scenarios.md))
+  - Fills up the persistent volumes, up to a givien percentage, used by the pod for the specified duration
```
Comment (chaitanyaenr) [inline docs/index.md:167]: s/givien/given

File: docs/index.md [hunk 3]
```diff
@@ -194,6 +199,7 @@ Let’s take a look at few recommendations on how and where to run the chaos tes
   - Chaos Engineering Without Observability ... Is Just Chaos
   - Make sure to have logging and monitoring installed on the cluster to help with understanding the behaviour as to why it’s happening. In case of running the tests in the CI where it’s not humanly possible to monitor the cluster all the time, it’s recommended to leverage Cerberus to capture the state during the runs and metrics collection in Kraken to store metrics long term even after the cluster is gone.
   - Kraken ships with dashboards that will help understand API, Etcd and OpenShift cluster level stats and performance metrics.
+  - Pay attention to Prometheus alerts. Check they are firing as expected.
```
Comment (chaitanyaenr) [inline docs/index.md:202]: *check if they are firing as expected

**Review by chaitanyaenr — COMMENTED** [review]
LGTM other than minor nits @agullon.

File: docs/index.md (hunk 1 above)
Comment (agullon) [reply docs/index.md]: fixed

File: docs/index.md (hunk 2 above)
Comment (agullon) [reply docs/index.md]: fixed

File: docs/index.md (hunk 3 above)
Comment (agullon) [reply docs/index.md]: fixed

**Review by chaitanyaenr — APPROVED** [review]
LGTM!

**Review by paigerube14 — APPROVED** [review]
LGTM

**Pull request merged**